lib/compiler/src/beam_asm.erl

```diff
@@ -186,6 +188,7 @@ build_file(Code, Attr, Dict, NumLabels, NumFuncs, Abst, SourceFile, Opts) ->
     Essentials = finalize_fun_table(Essentials1, MD5),
     {Attributes,Compile} = build_attributes(Opts, SourceFile, Attr, MD5),
     AttrChunk = chunk(<<"Attr">>, Attributes),
+    RecsChunk = chunk(<<"Recs">>, term_to_binary({raw_records_v1, Records})),
```

Comment:
Should we only include the chunk only if there are actual records? Records are mostly an Erlang construct so I wonder if we should keep the chunk optional?